### PR TITLE
converted 3 classes to kotlin

### PR DESCRIPTION
**Pull request by nikit19**

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/model/Attributes.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+/**
+ * Created by harsimar on 20/05/18.
+ */
+
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy::class)
+data class Attributes(var identifier: String? = null,var name: String? = null,var organizerDescription: String? = null,var startsAt: String? = null,var originalImageUrl: String? = null,var description: String? = null )
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/model/Attributes.kt]: Move each var per line

File: app/src/main/java/org/fossasia/openevent/general/model/Login.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package org.fossasia.openevent.general.model
+
+/**
+ * Created by nikit on 11/5/18.
+ */
+data class Login(var email: String? = null, var password: String? = null)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/model/Login.kt]: Make val and non nullable

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Remove created by template

**Comment by nikit19** [comment]
@iamareebjamal Please review. Done with requested changes

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**